cmd/mev-relay-proxy/main.go

```
@@ -63,7 +65,7 @@ func main() {
 		}
 	}()
 	// init service and server
-	svc := api.NewService(l, _BuildVersion, *nodeID, *authKey, clients...)
+	svc := api.NewService(l, _BuildVersion, _SecretToken, *nodeID, *authKey, clients...)
```

Add validation for the `_SecretToken`. If it is empty, we should fail the application from starting. This helps us to avoid the scenario where we might mistakenly assume the built binary contains the token, potentially missing it. It's better to catch this condition here and fail early rather than connecting and seeing issues later

---

Added the validation.